tests/test_confidence.py

```diff
@@ -5,6 +5,7 @@
 
 import lmfit
 from lmfit_testutils import assert_paramval
+from scipy.stats import f
```

Comment:
this third-party import should ideally be after ```import pytest```, followed by a newline and then lastly the ```lmfit``` related imports